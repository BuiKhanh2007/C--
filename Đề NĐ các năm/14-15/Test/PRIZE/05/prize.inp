93458888
68511112
