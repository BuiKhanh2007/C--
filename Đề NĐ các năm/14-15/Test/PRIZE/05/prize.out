161970000
9